client/components/Reports/AriaSupportTable.jsx

```diff
@@ -6,28 +6,32 @@ import { Table } from 'react-bootstrap';
 
 const columnName = ({ atName, browserName }) => `${atName} and ${browserName}`;
 
-const AriaSupportTable = ({ ariaHtmlFeaturesMetrics }) => {
-  console.log(ariaHtmlFeaturesMetrics);
+const AriaSupportTable = ({ ariaHtmlFeaturesMetrics, headingText }) => {
```

Comment:
Not encouraging it too hard, but if you think it would clean up duplication - now that you added `headingText` I think you could pass only `ariaFeaturesByAtBrowser` or `htmlFeaturesByAtBrowser` to a `dataRows` param here and possibly delete the other `HtmlSupportTable`